.gitpod.yml

```diff
@@ -44,6 +44,22 @@ tasks:
           exit 0
     # This task takes care of configuring your workspace so it can manage and interact
     # with preview environments.
+    - name: Dogfood Check
```

Comment:
I've fixed this by making "Dogfood" task the "latest" so that it's shown on top. If canceled, the terminal will close. 

I could not test this yet due to https://gitpod.slack.com/archives/C01KGM9BH54/p1679580560373679.